### PR TITLE
Create relation between grid company & grid areas

### PR DESCRIPTION
**Pull request by MrDach**
## Description

Created model + DB config and migration

## References

* #539

### CODE REVIEW

File: source/Energinet.DataHub.MeteringPoints.Domain/Actors/Actor.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        private Actor()
+        {
+        }
+#pragma warning restore 8618
+
+        public ActorId Id { get; }
+
+        public string IdentificationNumber { get; }
+
+        public ActorType ActorType { get; }
```
Comment (cmgrd) [inline source/Energinet.DataHub.MeteringPoints.Domain/Actors/Actor.cs:39]: `IdentificationType`

File: source/Energinet.DataHub.MeteringPoints.Domain/Actors/ActorType.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+// limitations under the License.
+
+using Energinet.DataHub.MeteringPoints.Domain.SeedWork;
+
+namespace Energinet.DataHub.MeteringPoints.Domain.Actors
+{
+    public class ActorType : EnumerationType
+    {
+        public static readonly ActorType GLN = new ActorType(0, nameof(GLN));
+        public static readonly ActorType EIK = new ActorType(1, nameof(EIK));
```
Comment (cmgrd) [inline source/Energinet.DataHub.MeteringPoints.Domain/Actors/ActorType.cs:22]: `EIC`

File: source/Energinet.DataHub.MeteringPoints.Domain/Actors/Actor.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections.ObjectModel;
+
+namespace Energinet.DataHub.MeteringPoints.Domain.Actors
+{
+    public class Actor
```
Comment (kft) [inline source/Energinet.DataHub.MeteringPoints.Domain/Actors/Actor.cs:19]: In this domain (Metering Points) the only market actor we are dealing with is the "Grid operator" (DSO). Maybe it is more  intent revealing to use a more specific name for this concept ? "Actor" sounds quite generic.

Comment (MrDach) [reply source/Energinet.DataHub.MeteringPoints.Domain/Actors/Actor.cs:19]: Closing this as per agreement on standup that Actor is the term to use. "Player" has been used previously but apparently that wasn't well received.

**Pull request closed without merging**